docs/content.zh/docs/core-concept/transform.md

```diff
@@ -39,10 +39,25 @@ To describe a transform rule, the following parameters can be used:
 | primary-keys | Sink table primary keys, separated by commas       | optional          |
 | partition-keys | Sink table partition keys, separated by commas       | optional          |
 | table-options | used to the configure table creation statement when automatically creating tables | optional          |
+| converter-after-transform | used to add a converter to change DataChangeEvent after transform                 | optional          |
 | description  | Transform rule description | optional          |
 
 Multiple rules can be declared in one single pipeline YAML file.
 
+## converter-after-transform
+
+`converter-after-transform` is used to change the DataChangeEvent after other transform. The available values of this options are as follows.
+- SOFT_DELETE: The delete event will be converted as an insert event. This converter should be used together with the metadata `__data_event_type__`. Then you can implement the soft delete.
+
+For example, the following transform will not delete data when the delete event happens. Instead it will update the column `op_type` to -D in sink when deleting data.
```

Comment:
```suggestion
For example, the following transform will not delete data when the delete event happens. Instead it will update the column `op_type` to -D in sink, and transform it to an insert record.
```